tests/pouchdb-rewrite/features.js

```diff
@@ -248,15 +248,15 @@ describe('sync rewrite tests', () => {
     })
 
     .then((error) => {
-      error.status.should.equal(401);
+      error.status.should.equal(404);
```

Comment:
@marten-de-vries would you know why this change and the one line 259 are needed when removing `pouchdb-seemsless-auth`? Is that concerning? (I haven't had time to investigate).

Reply:
Ah, so this was why ``pouchdb-seamless-auth`` was included. The changed test is essentially worthless. The test essentially tests parts of ``pouchdb-route`` and ``pouchdb-seamless-auth``. Both are relatively useless from an ``express-pouchdb`` perspective, but the first is essential for maintaining the current ``pouchdb-rewrite`` API.